pkg/projects/Microsoft.NETCore.UniversalWindowsPlatform/Microsoft.NETCore.UniversalWindowsPlatform.pkgproj

```diff
@@ -30,6 +30,11 @@
       <TargetFramework>.NETCore50</TargetFramework>
     </Dependency>
 
+    <Dependency Include="Microsoft.Net.UWPCoreRuntimeSdk">
```

Comment:
This is the same mechanism which Microsoft.NETCore.DotNetHostResolver depends on Microsoft.NETCore.DotNetAppHost. See [pkg/projects/Microsoft.DotNetHostResolver/Microsoft.NETCore.DotNetHostResolver.pkgproj](https://github.com/dotnet/core-setup/blob/master/pkg/projects/Microsoft.NETCore.DotNetHostResolver/Microsoft.NETCore.DotNetHostResolver.pkgproj#L5):

```
  <ItemGroup>
    <Dependency Include="Microsoft.NETCore.DotNetAppHost">
      <Version>$(AppHostFullVersion)</Version>
    </Dependency>
  </ItemGroup>
```

Is this the correct way to declare the project to project dependencies?